web/src/components/ResetSchemaButton.tsx

```diff
@@ -82,12 +86,13 @@ export const ResetSchemaButton = (props: Props) => {
               </Button>
               <Button
                 disabled={resettingSchema}
-                colorScheme={initialized ? "red" : "green"}
+                background={useColorModeValue("#ECE8FD", "#2F206E")}
+                color={useColorModeValue("#553ACF", "#ECE8FD")}
                 onClick={onResetSchema}
                 ml={3}
               >
                 {resettingSchema ? (
```

Comment:
I'll stop mentioning them, but you have a lot of uses of ternary operators that should be rewritten. I see that this one kinda existed before, but we should still get rid of them, especially if they are chained like this one (i.e. multiple ones together).

Reply:
Have resolved all ternary operations